### ISSUE TITLE
Decision tree to estimate boman index from 3 parameters

### ISSUE THREAD

**Issue by DeadlineWasYesterday** [DeadlineWasYesterday/ML#7]
Find the file "druginfo.csv" on the Michael-DecTreeReg branch. 

The data is collected from an antibacterial peptide database. There are 4 parameters. We are going to try and use decision tree regression to see if we can estimate the last parameter (boman index) from the rest. 

Remember to split the dataset into training and test sets. Train your algorithm with your training data and see how well it the predictions appear for the test set.
